packages/grid_client/tests/zos/deployments.test.ts

```diff
@@ -0,0 +1,827 @@
+import { Keyring } from "@polkadot/keyring";
+import { waitReady } from "@polkadot/wasm-crypto";
+import { plainToInstance } from "class-transformer";
+import { validate } from "class-validator";
+import { default as md5 } from "crypto-js/md5";
+
+import { FLISTS } from "../../src/helpers/flists";
+import { ComputeCapacity } from "../../src/zos/computecapacity";
+import { Deployment, KeypairType, Signature, SignatureRequest, SignatureRequirement } from "../../src/zos/deployment";
+import { Workload, WorkloadTypes } from "../../src/zos/workload";
+import { MyceliumIP, Zmachine, ZmachineNetwork } from "../../src/zos/zmachine";
+
+const TEST_CONSTANTS = {
+  TWIN_ID: 123,
+  CONTRACT_ID: 456,
+  EXPIRATION: 1000000000,
+  METADATA: "test_metadata",
+  DESCRIPTION: "test_description",
+  WEIGHT_REQUIRED: 10,
+  WEIGHT: 5,
+
+  WORKLOAD_NAME: "test_workload",
+  WORKLOAD_METADATA: "workload_metadata",
+  WORKLOAD_DESCRIPTION: "workload_description",
+
+  CPU: 1,
+  MEMORY: 1024 * 1024 * 1024, // 1GB in bytes
+  SIZE: 10 * 1024 * 1024, // 10MB in bytes
+  PUBLIC_IP: "10.0.0.1",
+  NETWORK_NAME: "test_network",
+  NETWORK_SEED: "test_seed",
+
+  MNEMONIC: "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about",
```

Comment:
can we just import the mnemonics used in config.json instead of using these static dummy mnemonic?